go-controller/pkg/cni/types/types.go

```diff
@@ -32,6 +32,11 @@ type NetConf struct {
 	// they are originally created - e.g. a KubeVirt VM's migration, or
 	// restart.
 	AllowPersistentIPs bool `json:"allowPersistentIPs,omitempty"`
+	// DisablePortSecurity is valid on both localnet / layer topologies.
+	// It allows the admin to opt out of port security, allowing nested
+	// virtualization scenarios, where traffic from multiple MAC addresses
+	// (bridged VMs) can egress over the OVN port
+	DisablePortSecurity bool `json:"disablePortSecurity,omitempty"`
```

Comment:
maybe call this `promiscMode` ? That seems more generic / aligned w/ other CNIs (for instance [bridge](https://github.com/containernetworking/plugins/blob/0259301ae2eab8cb4f5755d790c29cb869765f8a/plugins/main/bridge/bridge.go#L57)).

Reply:
disregard this comment; I had misunderstood the behavior of the `unknown` value on the addresses field.

Copying from the [OVN nb](https://man7.org/linux/man-pages/man5/ovn-nb.5.html#Logical_Switch_Port_TABLE) man page:
```
                     This indicates that the logical port has an unknown
                     set of Ethernet addresses. When an OVN logical
                     switch processes a unicast Ethernet frame whose
                     destination MAC address is not in any logical
                     port’s addresses column, it delivers it to the port
                     (or ports) whose addresses columns include unknown.
```